docs/docs/getting-started/04-examples/04-jtc-example.md

```diff
@@ -0,0 +1,202 @@
+        call_service:
+          controller: joint_trajectory_controller
+          hardware: hardware
+          service: set_trajectory
+          payload: |-
+            {
+              frames: [start, waypoint_1, waypoint_2, waypoint_3, start], 
+              times_from_start: [2.0, 4.0, 6.0, 8.0, 10.0]
+            }
+  edges:
+    hardware_hardware_joint_trajectory_controller_has_trajectory_succeeded_on_stop_on_stop:
+      path:
+        - x: 440
+          y: 780
+        - x: 440
+          y: 700
+```
+</details>
+
+The application graph should show a hardware interface with a Joint Trajectory controller and an event trigger, as shown
+below:
+
+<div class="text--center">
+  <img src={jtcExampleOverview} alt="Simple JTC configuration" />
+</div>
+
+## The example explained
+
+The application starts by loading the Robot State broadcaster and Joint trajectory controllers for the generic six-axis
+robot. You may already play the application.
+
+After you press play, switch to the 3D view using the corresponding button on the top right of your AICA Studio. You
+should see your robot in its default joint configuration, along with 4 Cartesian frames named
+**start, waypoint_1, waypoint_2, and waypoint_3** that form a triangle. As you may have guessed, this application
+controls the robot such that it traverses the Cartesian frames in that order. 
+
+<div class="text--center">
+  <img src={jtcExampleFrames} alt="Cartesian frames for JTC" />
+</div>
+
+For now, let us go back to the graph view. Once there, turn your attention to the trigger button that connects to the
+`Set trajectory` service of the Joint Trajectory controller. Click on the gear icon on the edge that connects the button
+to the controller. You will see the following service payload string:
+
+```yaml
+{
+  frames: [start, waypoint_1, waypoint_2, waypoint_3, start], 
+  times_from_start: [2.0, 4.0, 6.0, 8.0, 10.0]
+}
+```
+
+This message defines 2 of the variables that JTC can use in order to execute a trajectory, namely: 
+
+1. `frames`: a list of the Cartesian frames that you would like to traverse, in the order that they should be visited.
+2. `times_from_start`: the amount of time (in seconds) JTC should take to move to each frame.
+ 
+Naturally, the two vectors need to be of the same size. Also, notice how the payload frame names correspond to the ones
+available in the 3D view (or yaml application) to anticipate what the trajectory execution will look like.
+
+## Executing the trajectory
+
+Assuming you have already pressed play and you are currently in the graph view, go ahead and press the trigger button to
+start the execution of the trajectory. Switch to the 3D view to see the robot moving through the waypoints.
+
+<div class="text--center">
+  <img src={jtcExampleRun} alt="JTC in action" />
+</div>
+
+That is all you need to move your robot along predefined poses! Once the trajectory is executed, a predicate is set and
+the application will automatically stop.
```

Comment:
I'd rather have the application loop this to be honest, for a user that is not as fast as us with controls it might give the chance to see the robot in action for a bit more.

Reply:
The JTC guide will show how to do that (a bit differently), but we could just connect the predicate to drive new repetitions, that should be fairly straightforward even for beginners